src/Resources/contao/templates/forms/form_captcha.html5

```diff
@@ -15,4 +15,16 @@
 
   <input type="text" name="<?= $this->name ?>" id="ctrl_<?= $this->id ?>" class="captcha mandatory<?php if ($this->class) echo ' ' . $this->class; ?>" value=""<?= $this->getAttributes() ?>>
   <span class="captcha_text<?php if ($this->class) echo ' ' . $this->class; ?>"><?= $this->getQuestion() ?></span>
+
+  <?php if (!$this->hasErrors()): ?>
+    <div style="display:none">
+      <label for="ctrl_<?= $this->id ?>_hp">Do not fill out this field</label>
```

Comment:
Yes, the PR wasn’t completely finished, thats why I added [RFC], not [RTM] :)
Because there is a `display:none` div directly around it, no user and no screenreader user will see that in 99.9999% of cases.
But yes, we should change it to `Do not fill in this field` and translate it.